test/k8s/client_test.exs

```diff
@@ -1,4 +1,78 @@
 defmodule K8s.ClientTest do
   use ExUnit.Case, async: true
   doctest K8s.Client
+
+  defmodule IntegrationTest do
+    use ExUnit.Case, async: false
+
+    def conn() do
+      conn =
+        "KUBECONFIG"
```

Comment:
Im not a fan of using KUBECONFIG as the test env var... should replace w/ K8S_KUBECONFIG or something.